lib/chroner/administrative.ex

```diff
@@ -16,7 +16,7 @@ defmodule Chroner.Administrative do
   # --------------------------------------------------------------------
 
   @doc "Fetches all doctors resources."
-  @spec doctors_list(client) :: {:ok, Doctor.t()} | error
+  @spec doctors_list(client) :: {:ok, list(Doctor.t()) | []} | error
```

Comment:
Use `[Doctor.t]` instead of `list(Doctor.t)`

Reply:
> In general it looks good. I don't like this commit though:
> 
> ![image](https://user-images.githubusercontent.com/202498/77319968-080a0480-6d10-11ea-9c52-56379081990d.png)
> 
> It packs too many things into a single commit. It could have been easily split across three commits:
> 
>     * Use ExVcr for testing
> 
>     * Add recorded casettes
> 
>     * Update readme (which would be combined with the last commit)

Yes I knew you're going to get heart attack for this. I was a little bit lazy.